### ISSUE TITLE
[Feature request] In Home timeline, hide posts from people in pinned lists

### ISSUE THREAD

**Issue by Cassolotl** [mastodon/mastodon#18151]
### Pitch

If someone has pinned a list, they should be able to hide posts that appear in that list from appearing in the Home timeline as well.

This would serve a similar/parallel function to the "remote only" filter in the federated timeline - allowing users to not have to look at the same toot twice.

### Motivation

[@seedlingattempt@kolektiva.social](https://kolektiva.social/@seedlingattempt) asked for it, here: https://kolektiva.social/@seedlingattempt/108208670474385142

>"For accounts that you add to lists and then pin in column view, can you remove those accounts from the home view?"

The [workaround](https://github.com/mastodon/mastodon/issues/7876#issuecomment-399695891) is:

>"ideally people who use lists would also be able to exlude accounts from their Home timeline, but in the meantime you can create a list called "Home", add people to it, and then unpin the actual home timeline"

However, this means that someone has to manually add people to the home list every time they follow someone. This feature request would save them a lot of effort.

**Comment by darkphoenix** [comment]
This is actually a really neat idea, I was just discussing lists with someone and this seemed like an obvious thing.  Either pin it, or have an extra switch for it?  I was going to make an issue for the latter, but it seems you beat me to it.

**Comment by matt22207** [comment]
+1

Seems like there should be an account setting to de-duplicate toots across ALL columns, in some priority order such as: simple left-to-right priority.. or something smarter like Pinned Lists (leftmost only if a toot appears in multiple Pinned Lists), Home, Local, Federated.

**Comment by bixfrankonis** [comment]
> This is actually a really neat idea, I was just discussing lists with someone and this seemed like an obvious thing. Either pin it, or have an extra switch for it? I was going to make an issue for the latter, but it seems you beat me to it.

The latter version (switch vs. pin), which personally I prefer, exists already in the Hometown fork, FWIW.

**Comment by jumase** [comment]
I agree on adding a swith to Lists that allows users to hide publications that already are on each list. I mean hide those toots on the Home timeline, to avoid duplicates. It would be a very useful feature. That way Lists would become more interesting.

**Comment by noellabo** [comment]
![fedibird_follow_dialog2](https://user-images.githubusercontent.com/28195220/180905641-2565a791-8488-49d8-8478-7c0ee4b2dc8e.png)
This is the Fedibird follow dialog. Select the target you want to feed from the Home and List.
In this case, the feed is only to the list (Others), and the feed to Home is disabled.
(don't mind the square-rss icon, it's a different feature)

Fedibird solves it this way. How do you like it?
(Fedibird is my Mastodon fork)

**Comment by rfkat** [comment]
> Select the target you want to feed from the Home and List.

Nice. Simple, with maximum freedom - can solve all issues I've seen so far.

**Comment by jumase** [comment]
> Fedibird solves it this way. How do you like it? (Fedibird is my Mastodon fork)

Hey @noellabo that's awesome!

**Comment by psyciknz** [comment]
Was adding a very similar feature suggestion to be able to mute from the home TL (only) a followed account.   But ability to set a list to say only show here, and remove it from Home TL is a much cleaner option.

**Comment by HilaryBaumann** [comment]
I want to add a little bit of extra info from a different perspective but similar to what everyone is saying here:

Before I found out that filters could filter an entire person without unfollowing them, I ended up unfollowing someone and trying to add them to a list and couldn't.

Now I have to wait for them to approve me following them again to even add them to a list. But their content is public AND viewable in the timeline while waiting for a follow approval. 

I am now in limbo where I have filtered this person (because if I don't they are 90% of my timeline) and shoved the rss of their account into my feedreader for the moment...because that's the only workaround I could think of to fix the fact that I can't put them in a list if they are a) waiting follow approval b) not followed.

For another example: if things like Weather or local Traffic accounts end up showing up on Mastodon like they have on other platforms, I don't really need to keep up with them on an rss level but when say there's something going on on that particular topic I want to see their recent posts. That's where I want to be able to use the lists. Do I need to follow these accounts and have them in my timeline? No. I just need to be able to quickly reference them as a group at certain times. Not have their firehose filling my timeline or my feedreader.

I hope what I'm saying is clear enough or helps explain a use case and what's not working well as-is.
